README.md

```diff
@@ -45,11 +45,9 @@ file_path_or_url = 'path/to/file.mp4'
 # process of fixing this though.
 player = OMXPlayer(file_path_or_url)
 
-# The player will initially be paused
-
-player.play()
-sleep(5)
 player.pause()
```

Comment:
Why have you swapped round these lines? Does the player no longer pause initially? If so, that's a bug, I don't want to change the default behaviour thus breaking backwards compatibility.